default.conf

```diff
@@ -2,6 +2,13 @@ server {
     listen 8080;
     root /usr/share/nginx/html/;
     index index.html;
+    error_page 403 /404.html;
+    location = /404.html {
+        internal;
+    }
+    location / {
```

Comment:
duplicated, already on line 12

Reply:
Fixed, I got confused when trying to fix the merge conflict I didn't understand.